feature/results/src/main/java/com/android/developers/androidify/customize/CustomizeExportScreen.kt

```diff
@@ -453,10 +453,10 @@ fun CustomizeExportPreviewLarge() {
         AnimatedContent(true) { targetState ->
             targetState
             CompositionLocalProvider(LocalNavAnimatedContentScope provides this@AnimatedContent) {
-                val bitmap = ImageBitmap.imageResource(R.drawable.placeholderbot)
+                val imageUri = ("android.resource://com.android.developers.androidify.results/" + R.drawable.placeholderbot).toUri()
```

Comment:
This seems like it could fail if we changed any location of the placeholderbot image. Is there a way to just use something like Context.fromResources() or of that similarity?